packages/devextreme/js/__internal/.eslintrc.js

```diff
@@ -66,8 +66,13 @@ module.exports = {
                 'no-underscore-dangle': 'off',
                 'no-console': ['error', { 'allow': ['warn', 'error'] }],
                 'class-methods-use-this': 'off',
+                'spellcheck/spell-checker': 'off',
                 '@typescript-eslint/ban-ts-comment': 'off',
                 '@typescript-eslint/prefer-ts-expect-error': 'off',
+                '@typescript-eslint/no-unsafe-return': 'off',
+                '@typescript-eslint/explicit-module-boundary-types': 'off',
+                '@typescript-eslint/explicit-function-return-type': 'off',
+                '@typescript-eslint/no-dynamic-delete': 'off',
```

Comment:
let's disable eslint checks with `// eslint-disable-next-line ...` in specific files